wikiquote/langs/eu.py

```diff
@@ -0,0 +1,23 @@
+from .. import utils
+
+WORD_BLACKLIST = ["Iturria:", "Jatorrizkoan ", "Testuingurua:"]
+MAIN_PAGE = "Azala"
+HEADINGS = ["kanpo loturak", "erreferentziak"]
+
+
+def extract_quotes(tree, max_quotes):
+    q_lst = utils.extract_quotes_li(tree, max_quotes, HEADINGS, WORD_BLACKLIST)
+    return [utils.remove_credit(q) for q in q_lst]
+
+
+def qotd(html_tree):
+    try:
+        tree = html_tree.get_element_by_id("mf-qotd")
+
+        selector = "div/div/table/tbody/tr"
+        raw_quote = tree.xpath(selector)[0].text_content().split("~")
+        quote = raw_quote[0].strip()
+        author = raw_quote[1].strip()
+        return quote, author
+    except KeyError:
+        return "Ez dago eguneko aipurik", ""
```

Comment:
With these types of errors becoming so common (the "citation of the day" pages sometimes change their layout) I think I will change his behaviour in the near future. I will probably create a new exception class that is raised when there was a layout-dependant problem when extracting the QOTD, for any language.

Reply:
You're welcome! In the case of Basque it was totally necessary because there is no quote of the day... But the same mechanism could be applied to the rest of the languages (with possibly a express-made exception), as you say.